src/main/java/org/codehaus/mojo/taglist/FileAnalyser.java

```diff
@@ -124,6 +134,8 @@ public FileAnalyser( TagListReport report, List<TagClass> tagClasses )
         locale = report.getLocale();
         noCommentString = report.getBundle().getString( "report.taglist.nocomment" );      
         this.tagClasses = tagClasses;
+        this.includes = report.getIncludes();
+        this.excludes = report.getExcludes();
```

Comment:
report should change methods, see below.

Reply:
@bmarwell done